build.gradle

```diff
@@ -25,6 +25,7 @@ repositories {
     	name = "Cloth+REI"
     	url = "https://maven.shedaniel.me/"
     }
+	maven { url = "https://www.cursemaven.com" }
```

Comment:
Oh, one last change before I'll download and build. 
I'd rather have the official maven repos, not cursemaven.
Please revert to the previous format and use the gradle.properties file to choose the correct version for dependencies

Reply:
I think gradle takes care of any inherited dependency, that should not be a problem. As for REI, I think they moved to another repo, so newer versions will be there. I tried in my other mod, extra alchemy, and it downloaded fine, if you give me a couple minutes I'll find the correct link

Reply:
actually, the repo is still the same, the project however changed id, and was split in api and implementation. The new dependency code block is 
```gradle

modCompileOnly("me.shedaniel:RoughlyEnoughItems-api-fabric:${project.rei_version}") {
	exclude module: 'fabric-api'
}

modRuntime("me.shedaniel:RoughlyEnoughItems-fabric:${project.rei_version}") {
	exclude module: 'fabric-api'
}
```